frontend/src/components/InputElement.vue

```diff
@@ -69,6 +80,9 @@ const value = computed({
     :id="id"
     v-model="value"
     v-bind="attributes"
-    :validation-error="validationError"
+    v-model:validation-error="validationError"
   />
+  <div class="h-16 label-03-reg text-red-800">
```

Comment:
Maybe we can add a comment here that this is the temporary location and should be refactored once we talked about error handling soon? Because now there is two locations, which might be confusing to developers coming across this.

Reply:
done